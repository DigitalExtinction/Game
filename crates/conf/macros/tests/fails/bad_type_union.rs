use conf_macros::Config;

#[derive(Config)]
union TestConfig {
    foo: u32,
    bar: u32,
}

fn main() {}
