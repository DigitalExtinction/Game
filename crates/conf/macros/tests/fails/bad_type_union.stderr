error: expected a struct, not a union
 --> tests/fails/bad_type_union.rs:4:1
  |
4 | union TestConfig {
  | ^^^^^
